openapi/tools/merge_openapi.py

```diff
@@ -0,0 +1,144 @@
+import sys
+import argparse
+import yaml
+from glob import glob
+from openapi_spec_validator import validate_spec
+from openapi_spec_validator.readers import read_from_filename
+from pathlib import Path
+
+global_had_load_error = False
+
+def validate_openapi(filepath):
+    spec_dict = read_from_filename(filepath)[0]
+    validate_spec(spec_dict)
+
+
+def yaml_loader(filepath):
```

Comment:
It would be nice to `trim()` all strings on load, so we don't get ugly trailing newlines. I don't know if it's possible to honor all the text wrapping directives though.

```
      summary: 'Enumerate resources in a workspace. The visible resources depend on
        the permissions of the caller.
        These are presented sorted by ascending resource name. The offset and limit
        parameters allow paging
        through the results. You can also filter by a resource type and by a stewardship
        type.
        '
```

Reply:
My thought was to standardize on one text wrapping directive that makes it through this transition the best. There are some controls on the generation side that might help, too. It will require some investigation. I made PF1378 to track this.